app/controllers/integrations/feature_flags_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Integrations
+  class FeatureFlagsController < IntegrationsController
+    def index
+      feature_flags = FeatureFlag::FEATURES.reduce({}) do |features, feature_name|
+        features[feature_name] = {
+          name: feature_name.humanize,
+          active: FeatureFlag.active?(feature_name),
+        }
+
+        features
+      end
+
+      render json: feature_flags
```

Comment:
Should we include the environment name in the response? Bit easier for callers

Reply:
Done!